storage/innobase/xtrabackup/doc/source/glossary.rst

```diff
@@ -4,136 +4,199 @@
 
-  .. code-block:: text
+      .. code-block:: text
 
-      [mysqld]
-      innodb_log_group_home=/var/lib/mysql
+         [mysqld]
+         innodb_file_per_table
 
-LSN
- Each InnoDB page (usually 16kb in size) contains a log sequence number (LSN). The LSN is the system version number for the entire database. Each
- page's LSN shows how recently it was changed.
+      or start the server with ``--innodb_file_per_table``.
 
-MyISAM
- Previous default storage engine for *MySQL* for versions prior to 5.5. It
- does not fully support transactions but in some scenarios may be faster
- than `InnoDB`. Each table is stored on disk in 3 files:
- `.frm`, `.MYD`, `.MYI`.
+   innodb_log_group_home_dir
+     Specifies the location of the |InnoDB| log files:
 
+      .. code-block:: text
 
-my.cnf
-  This file refers to the database server's main configuration file. Most Linux distributions have this file at `/etc/mysql/my.cnf` or `/etc/my.cnf`, but the location and name depends on the particular installation. Note that this is not the only way to configure the server, and some systems rely only on the command options to start the server and set the defaults values.
+         [mysqld]
+         innodb_log_group_home=/var/lib/mysql
 
-UUID
-  A Universally Unique Identifier (UUID) which uniquely identifies the state and the sequence of changes on a node. The 128-bit UUID is a classic DCE UUID Version 1 and is based on current time and MAC address. In Galera, it is based on the generated pseudo-random addresses ("locally administered" bit in the node address (in the UUID structure) is always equal to unity).
+   logical backup
+      A backup which contains a set of SQL statements. The statements can be used to recreate the databases. 
+
+   LSN
+     Each InnoDB page (usually 16kb in size) contains a log sequence number, or
+     LSN. The LSN is the system version number for the entire database. Each
+     page's LSN shows how recently it was changed.
+
+   my.cnf
+     The database server's main configuration file. Most
+     Linux distributions place it as :file:`/etc/mysql/my.cnf` or
+     :file:`/etc/my.cnf`, but the location and name depends on the particular
+     installation. Note that this is not the only way of configuring the
+     server, some systems does not have one and rely on the command
+     options to start the server and its defaults values.
+
+   MyISAM
+     Previous default storage engine for |MySQL| for versions prior to 5.5. It
+     doesn't fully support transactions but in some scenarios may be faster
+     than :term:`InnoDB`. Each table is stored on disk in 3 files:
+     :term:`.frm`, :term:`.MYD`, :term:`.MYI`.
+  
+   physical backup
+     A backup that copies the data files.
+
+   point in time recovery
+     This method allows data to be restored into the state it was in any selected point of time.
```

Comment:
in any selected point of time -> at any selected point in time

Reply:
Condition may have other meanings.